From: Boyuan Yang <073plan@gmail.com>
Date: Mon, 17 Oct 2016 23:29:04 +0800
Subject: asemantools.pro: patch build instructions

---
 asemantools.pro | 2 --
 1 file changed, 2 deletions(-)

diff --git a/asemantools.pro b/asemantools.pro
index ccbfeb2..c315e17 100644
--- a/asemantools.pro
+++ b/asemantools.pro
@@ -31,8 +31,6 @@ qmlFiles.target = $$DESTDIR/../
 COPYFOLDERS += qmlFiles
 
 !win32-msvc* {
-    include (qmake/copyData.pri)
-    copyData ()
 }
 
 isEmpty(PREFIX) {
